gbols/fast-food
# #160720126 Endpoint to post an order.

**What does this PR do?**
It enables a user to post an order.

**Description of Task to be completed?**
- create the endpoint to enable a user to get a particular order.

**How should this be manually tested?**
- pull from this branch `ft-endpoint-post-order-160720126
`into a new branch.
- run npm install.
- the endpoint should look like this: `http://localhost:3000/api/v1/order`.
- run `npm test`.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
 #160720126 a user should be able to post an order.
Screenshots (if appropriate)
![screenshot from 2018-09-24 16-11-29](https://user-images.githubusercontent.com/27724644/45960681-a5e61b80-c014-11e8-9254-39cb6ede883f.png)
